react/miniproject/components/userdashboard/userDrawer.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import AppBar from 'material-ui/AppBar';
+import Drawer from 'material-ui/Drawer';
+import MenuItem from 'material-ui/MenuItem';
+import List from 'material-ui/List';
+import Divider from 'material-ui/Divider';
+import Subheader from 'material-ui/Subheader';
+import ListItem from 'material-ui/List/ListItem';
+
+class UserDrawer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { open: true };
+        this.handleToggle = this.handleToggle.bind(this);
+        this.handleClose = this.handleClose.bind(this);
+    }
+
+    handleToggle = () => {
+        this.setState({ open: !this.state.open });
+    };
+
+    handleClose = () => {
+        this.setState({ open: false });
+    };
+
+    render() {
+        return (
+            <MuiThemeProvider>
+                <div>
+                    <AppBar
+                        title="Shivani Mulay"
+                        onLeftIconButtonTouchTap={this.handleToggle}
+                    />
+                    <Drawer
+                        docked={false}
+                        width={200}
+                        open={this.state.open}
+                        onRequestChange={this.handleToggle}
+                    >
+                    <List>
+                        <Subheader><font color="#006064" size="4"><b>My Account</b></font></Subheader>
```

Comment:
Make file `color.js` and use colors from there by importing that file. It will help to change color at several places by making change at only one place.